elastic/kibana
# [Actionable Observability] Fix alerts' blank page in case of invalid query string

Implements #143641

## 📝 Summary
Fixes the alerts page crash when a wrong query is entered in the search bar query string.

![image](https://user-images.githubusercontent.com/12370520/201547177-b9d78bfe-545c-455a-b120-3455f8196607.png)

**Note**
I am working on tests but I will create a separate PR for that.

## 🧪 How to test
- Go to alerts / rule details page
- Enter an invalid query such as `{`, page should not crash and you should see a toast with a related error message
